test/Microsoft.Azure.WebJobs.Host.FunctionalTests/ILoggerTests.cs

```diff
@@ -124,7 +130,7 @@ public void Aggregator_Runs_WhenEnabled_AndFlushes_OnStop()
 
             // Add will be called 5 times. The default aggregator will ingore the 
             // 'Function started' calls.
```

Comment:
Now that you've changed this to ignore in the test, you can remove this comment.